juniper_official/interface/collect-physical-interface-stats.rule

```diff
@@ -0,0 +1,225 @@
+/*
+ * Collect Physical Interface trafficstatistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * One input control detection
+ *
+ *   1) input-ifd-name, is a regular expression that matches the interfaces
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all interfaces. Use something like 'ge.*' to match only gigabit
+ *      ethernet interfaces. 
+ */
+ healthbot {
+    topic interfaces {
+        rule collect-physical-interface-stats {
+            keys interface-name;
+            description "Collects IFD stats";
+            sensor interfaces-oc {
+                synopsis "Interface open-config sensor definition";
+                description "Interface open-config sensor definition";
+                open-config {
+                    sensor-name /interfaces/interface/state;
+                    frequency 60s;
+                }
+            }
+            field ifd-egress-bps {
+                formula {
+                    rate-of-change {
+                        field-name "$ifd-egress-octets";
+                        multiplication-factor 8;
+                    }
+                }
+                type integer;
+            }
+            field ifd-egress-octets {
+                sensor interfaces-oc {
+                    path /interfaces/interface/state/counters/out-octets;
+                }
+                type integer;
+                description "Interface egress octets counter";
+            }
+            field ifd-egress-packets {
+                sensor interfaces-oc {
+                    path /interfaces/interface/state/counters/out-pkts;
+                }
+                type integer;
+                description "Interface egress packets counter";
+            }
+            field ifd-egress-pps {
+                formula {
+                    rate-of-change {
+                        field-name "$ifd-egress-packets";
+                    }
+                }
+                type integer;
+            }
+            field ifd-ingress-bps {
+                formula {
+                    rate-of-change {
+                        field-name "$ifd-ingress-octets";
+                        multiplication-factor 8;
+                    }
+                }
+                type integer;
+            }
+            field ifd-ingress-octets {
```

Comment:
Can we use same naming convention as defined in below rule? At some stage if we want to combine both the rules, it will reduce the changes in business services. In this case it is "in-octets". Same goes with rest of the fields and IFL rule as well

https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/interface/check-interface-in-out-errors-traffic-state-flaps.rule